package.json

```diff
@@ -48,7 +48,7 @@
     "mocha-lcov-reporter": "^1.0.0",
     "mocha-tldr-reporter": "^1.0.0",
     "sinon": "^1.17.2",
-    "sinon-chrome": "^1.1.0",
+    "sinon-chrome": "^2.1.2",
```

Comment:
Hi. Can you revert this change?
I'll update this dependency in separate release